rust/otap-dataflow/crates/query-engine/src/pipeline/attributes.rs

```diff
@@ -280,4 +280,70 @@ mod test {
             )
         }
     }
+
+    #[tokio::test]
+    async fn test_delete_attributes() {
+        let result = exec_logs_pipeline(
+            "logs | project-away attributes[\"x\"]",
+            generate_logs_test_data(),
+        )
+        .await;
+        assert_eq!(
```

Comment:
nit: Maybe asserting that the result does not contain the deleted attribute is more easier to read the tests (in future by others!)